SDK/ViewControllers/Settings/SettingsManager.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  SettingsManager.swift
+//  iOS-SDK
+//
+//  Created by Alberto Huerdo on 2018. 12. 15..
+//  Copyright © 2018 SchedJoules. All rights reserved.
+//
+
+import Foundation
+
+class SettingsManager {
+    
+    enum SettingsType: String {
+        case language
+        case country
+        
+        var displayName: String {
+            switch self {
+            case .language:
+                return "Language"
+            case .country:
+                return "Country"
+            }
+        }
+        
+        var path: String {
+            switch self {
+            case .language:
+                return UserDefaultsKeys.Settings.language
+            case .country:
+                return UserDefaultsKeys.Settings.country
+            }
+        }
+        
+    } 
+    
+    static func get(type: SettingsType) -> SettingsObject {
+        guard let existingData = UserDefaults.standard.object(forKey: type.path) as? Data,
+            let existingObject = try? JSONDecoder().decode(SettingsObject.self, from: existingData) else {
+                
+                var defaultObject = SettingsObject()
```

Comment:
Original comment by @dmfs: "This part is the reason for making SettingsObject mutable I guess. Instead of making it mutable how about initializing it with the correct value?
https://github.com/schedjoules/ios-public-calendars-sdk/pull/19#discussion_r245274531

Reply:
All properties can be turned from var to let except fro icon, since it's not present in a settingsObject of type .language, and make it immutable would cause that the language cannot be stored in UserDefaults.
commit with the changes is https://github.com/schedjoules/ios-public-calendars-sdk/pull/23/commits/6bc3833865621d626d1da2788169d8bf549b9f2e